miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditStudyController.java

```diff
@@ -105,8 +106,23 @@ public Project populateProject(@PathVariable Long projectId) throws IOException
   public Collection<StudyType> populateStudyTypes() throws IOException {
     return studyService.listTypes();
   }
+  
+  @ModelAttribute("projects")
+  public Collection<Project> populateProjects() throws IOException {
+    return projectService.listAllProjects();
+  }
+
+  // put list of projects into Model object
```

Comment:
Does this comment go with the `populateProjects()` call just above?

Reply:
Leftover comment I forgot to remove - took it out in latest commit